### PR TITLE
Update permission system docs with new permissions

### PR DESCRIPTION
**Pull request by Ninjaclasher**

### CODE REVIEW

File: docs/site/permission_system.md [hunk 1]
```diff
@@ -79,17 +82,20 @@ The user can see all problems. However, they will *not* be able to edit problems
...
 #### `edit_all_problem` (Edit all problems)
-**Prerequisite perrmisions: `edit_own_problem`**
+**Prerequisite permissions: `edit_own_problem`**
 
 **Superceded permissions: `see_private_problem`, `edit_public_problem`, `view_all_submission`**
 
 The user can see and edit all problems on the admin site.
 
+#### `problem_full_markup` (Edit problems with full markup)
+The user will be able to edit the description of problems which offer full markup.
```
Comment (kiritofeng) [inline docs/site/permission_system.md:97]: Is the distinction between the limited and full markup included anywhere?

File: docs/site/permission_system.md [hunk 2]
```diff
@@ -79,17 +82,20 @@ The user can see all problems. However, they will *not* be able to edit problems
 The user can edit problems on the admin site, but only if they are explicitly set as author or curator. They will also be able to view submission source code for these problems.
 
 #### `edit_public_problem` (Edit public problems)
-**Prerequisite perrmisions: `edit_own_problem`**
+**Prerequisite permissions: `edit_own_problem`**
 
 The user can edit problems on the admin site, but only if the problem is public. They will also be able to view submission source code for these problems.
```
Comment (kiritofeng) [inline docs/site/permission_system.md:87]: Should clarify interaction between public and organization public

File: docs/site/permission_system.md [hunk 3]
```diff
@@ -54,6 +54,9 @@ The user can edit the `is_visible` field on the contest admin page.
 #### `contest_problem_label` (Edit contest problem label script)
 The user can edit the contest problem label script on the contest admin page.
 
+#### `lock_contest` (Change lock status of contest)
```
Comment (kiritofeng) [inline docs/site/permission_system.md:57]: Do we mention what locking a submission does?

**Comment by Ninjaclasher** [comment]
Updated the commit to reflect comments.

**Pull request merged**